x-pack/legacy/plugins/siem/public/pages/case/components/user_list/index.tsx

```diff
@@ -42,7 +42,7 @@ const renderUsers = (users: ElasticUser[]) => {
           <EuiFlexItem>
             <p>
               <strong>
-                <small>{username}</small>
+                <small data-test-subj="case-view-username">{username}</small>
               </strong>
```

Comment:
I like the feedback after clicking on email icon, I think that's a placeholder callback for now,  but still I'm not sure if it is fine to use `window.alert` on Line 53